components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -42,42 +43,75 @@ const VulnerabilitiesGrid = ({ onUpdate }) => {
   const updateGrid = newGrid => {
     setGrid(newGrid);
     onUpdate({
-      assets: Object.values(newGrid.assets),
-      vulnerabilities: Object.values(newGrid.vulnerabilities)
+      assets: Object.values(newGrid.assets).filter(a => a !== 'Other'),
+      vulnerabilities: Object.values(newGrid.vulnerabilities).filter(v => v !== 'Other')
```

Comment:
This could possibly be refactored to re-use the logic in both, to avoid repetition but it's pretty minimal anyway. So just something to consider...

```js
const excludeOther = item => item !== 'Other';
...
assets: Object.values(newGrid.assets).filter(excludeOther),
vulnerabilities: Object.values(newGrid.vulnerabilities).filter(excludeOther)
```